pkgs/applications/science/biology/bamtools/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, jsoncpp
+, pkg-config
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bamtools";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "pezmaster31";
+    repo = "bamtools";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-l2DmA4P1kPneTCL9YVACE6LcQHT0F+mufPyM69VkksE=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    jsoncpp
+    zlib
+  ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "C++ API & command-line toolkit for working with BAM data";
+    homepage = "https://github.com/pezmaster31/bamtools";
+    changelog = "https://github.com/pezmaster31/bamtools/releases/tag/${finalAttrs.src.rev}";
```

Comment:
```suggestion
    changelog = "https://github.com/pezmaster31/bamtools/releases/tag/v${finalAttrs.version}";
```

Reply:
Thanks for your review!

I'm not sure if this package is the same, but I prefer `src.rev` to `prefix + version` because sometimes they tag a release with a prefix once but not in another release.